LykkeApi2/Controllers/AssetsController.cs

```diff
@@ -24,14 +24,15 @@ public class AssetsController : Controller
         #endregion

```

Comment:
Please undo changes in this file. IClientAccountSettingsClient is more appropriate  than IClientAccountClient .